src/gui/static/src/app/components/layout/hardware-wallet/hw-passphrase-activation-dialog/hw-passphrase-activation-dialog.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { HwPassphraseActivationDialogComponent } from './hw-passphrase-activation-dialog.component';
+
+describe('HwPassphraseActivationDialogComponent', () => {
+  let component: HwPassphraseActivationDialogComponent;
+  let fixture: ComponentFixture<HwPassphraseActivationDialogComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ HwPassphraseActivationDialogComponent ],
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(HwPassphraseActivationDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
  it("should be created", () => {
```